src/Illuminate/Support/Number.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Illuminate\Support;
+
+use Illuminate\Support\Traits\Macroable;
+
+class Number
+{
+    use Macroable;
```

Comment:
I added the `Macroable` trait, as the `Str` class uses it. Let me know if we should keep it.

Reply:
We should. The Number helper has been requested many times before... This way people can add their functionality through packages before it's added to the core.

Reply:
Okay awesome! Resolving this conversation. Thanks :)